Casks/m/millie.rb

```diff
@@ -1,15 +1,21 @@
 cask "millie" do
-  version "2.1.2,100"
+  version "2.1.2"
   sha256 "1ec0830ae6daeafd0eefe335f8b005feacfb9b3af2b14ddddd6e33db13917123"
 
-  url "https://install.millie.co.kr/flutter/#{version.csv.first}/millie.dmg"
+  url "https://install.millie.co.kr/flutter/#{version}/millie.dmg"
   name "Millie"
   desc "Korean e-book store"
   homepage "https://www.millie.co.kr/"
 
   livecheck do
-    url "https://install.millie.co.kr/flutter/millie.dmg"
-    strategy :extract_plist
+    url "https://install.millie.co.kr/flutter/flutter_desktop_version.json"
+    strategy :json do |json|
+      json["versions"].each do |version, platforms|
+        next if platforms["macos"] != "prod"
+
+        break version if version.present?
+      end
```

Comment:
```suggestion
      json["versions"]&.map do |version, platforms|
        next if platforms["macos"] != "prod"

        version
      end
```

This change ensures that we retrieve the latest `prod` version if there is more than one.